conan/tools/google/bazel.py

```diff
@@ -77,4 +79,4 @@ def test(self, target=None):
         """
         if self._conanfile.conf.get("tools.build:skip_test", check_type=bool) or target is None:
             return
-        self._safe_run_command(f'bazel test {target}')
+        self._safe_run_command(f"{self._bazel_base_command} test {target}")
```

Comment:
I think this would be cleaner if it was something like:

```
self._safe_run_command(f"bazel test {self._get_startup_command_options()} {target}")
```

And the same for `build` and `clean`